<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addhotel</class>
 <widget class="QMainWindow" name="addhotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>蛋壳行记</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;等线&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bOK">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>申请创建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bCancel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cLoc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>选择地区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海淀区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>朝阳区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>丰台区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东城区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西城区</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lHName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lTel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;添加酒店&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;酒店名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;酒店位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;联系电话&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>bOK</zorder>
   <zorder>bCancel</zorder>
   <zorder>cLoc</zorder>
   <zorder>lTel</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lHName</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
